src/store/store.js

```diff
@@ -234,6 +234,8 @@ export default new Vuex.Store({
 			commit('setTables', [...tables])
 			return true
 		},
+		async transferTable({ state, commit, dispatch }, { tableId, newOwnerUserId, userId}) {
+		},
```

Comment:
I'm guessing this is whereI wanna call the backend code that does the ownership transfer. Not sure how to do that.